src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+require 'webmock/rspec'
+# WARNING: If you change owner tests make sure you uncomment this line
+# and start a test backend. Some of the Owner methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Webui::ObsFactory::StagingProjectsController, type: :controller, vcr: true do
+  let(:factory) { create(:project, name: 'openSUSE:Factory') }
+  let!(:factory_staging_a) { create(:project, name: 'openSUSE:Factory:Staging:A', description: '') }
```

Comment:
Whats the significance of the empty description?

Reply:
For nothing...but if I don't put any it fails. I will add some fake content to not have an empty string.